lib/src/bindings/structs.dart

```diff
@@ -13,6 +14,19 @@ class IDArray {
     _structPtr.elementAt(1).store(ids.length);
   }
 
+  List<int> _ids; // obx_id === uint64_t
+
+  IDArray.fromAddress(int address) {
```

Comment:
Seems like we should change `IDArray` to an `ffi.Struct`: see https://github.com/dart-lang/samples/blob/master/ffi/structs/structs.dart#L52
with the C struct looking like this: https://github.com/dart-lang/samples/blob/master/ffi/structs/c/structs.h#L11

`OBX_id_array` is defined here:
https://github.com/objectbox/objectbox-c/blob/master/include/objectbox.h#L319

Reply:
also, memory is not being reed using `obx_id_array_free()`

Reply:
I agree `Struct` would make this prettier. I already tried hacking this the `ffi.Struct` way, but for some reason I couldn't get it right. (wrong imports? wrong version?)
You could give it a try.

Reply:
`obx_id_array_free` done.